website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+
+    ## Bug fixes
+
+    - The IDE now displays an error message when the git repository is not reachable. Previously, it failed silently.
+    - The kebab menu is now visible when the invocation history drawer is open. Previously, it wasn't showing.
+    - DAGs are now updated/populated consistently. Previously, it occasionally failed.
+    - The purple highlight for DAG selection is now consistent across files. Previously, it was inconsistent.
+    - Users can now rename files back to their original name. Previously, this wasn't possible.
+    - The link to the IDE from the project setup page has been corrected.
+    - The IDE no longer has issues with single-space file names.
+    - Adding invalid characters in the sub-directory config no longer causes the IDE to fail.
+    - YML autocomplete triggers consistently now. Previously, it occasionally didn't trigger.
+    - Reverting single files now reloads the file contents in the tab. Previously, it didn't reload.
+    - The file tree no longer collapses on the first click when there is a project subdirectory defined.
+
+  </expandable>
+
+- <expandable alt_header='API updates'>
+
+    To make the API more scalable and reliable, we've implemented a maximum limit of `100` for all API requests to our `list` endpoints. If API requests exceed the maximum limit parameter of `100`, a user will receive an API error message.
+
+    This maximum limit applies to [multi-tenant instances](/docs/cloud/about-cloud/access-regions-ip-addresses) only, and _does not_ apply to single tenant instances.
+
+    Refer to the [Pagination](https://docs.getdbt.com/dbt-cloud/api-v2-legacy#section/Pagination) section for more information on this change. 
+
+    </expandable>
+
+## Feb 2023
+
+- <expandable alt_header='dbt Cloud IDE'>
+
+    To continue improving our [Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) experience, the dbt Labs team worked on fixing bugs, increasing reliability, and adding new features ✨.
+
+    Learn more about the [February changes](https://getdbt.slack.com/archives/C03SAHKKG2Z/p1677605383451109). 
+
+    ## New features 
+
+    - Support for custom node colors in the IDE DAG visualization
+    - Ref autocomplete includes models from seeds and snapshots
+    - Prevent menus from getting cropped (git controls dropdown, file tree dropdown, build button, editor tab options)
+    - Additional option to access the file menu by right-clicking on the files and folders in the file tree
+    - Rename files by double-clicking on files in the file tree and the editor tabs
+    - Right-clicking on file tabs has new options and will now open at your cursor instead of in the middle of the tab
+    - The git branch name above **Version Control** links to the repo for specific git providers
+        * Currently available for all [multi-tenant](/docs/cloud/about-cloud/access-regions-ip-addresses) instances using GitHub or GitLab providers 
```

Comment:
this is rendering weird fyi

![Screenshot 2024-05-15 at 12 44 18](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/44e2e7c1-efe7-497a-8f79-453861e902a0)